packages/build-utils/src/errors.ts

```diff
@@ -61,14 +61,14 @@ export function getPrettyError(obj: {
     }
 
     return new NowBuildError({
-      code: 'DEV_VALIDATE_CONFIG',
+      code: 'INVALID_LOCAL_CONFIG',
```

Comment:
Convention is usually "noun - verb" for error naming. How about this?

```suggestion
      code: 'VERCEL_CONFIG_INVALID',
```

Reply:
When I search this repo, I found many existing error codes that begin with `INVALID_` such as

```
invalid_build_specification
INVALID_DOMAIN
INVALID_DEPLOYMENT_ID
INVALID_ALIAS
invalid_email
invalid_function
INVALID_PIPFILE_LOCK 
```

Reply:
I'll change it to `invalid_vercel_config` [126695e](https://github.com/vercel/vercel/pull/8623/commits/126695e7a6b78e98afce808e7dc01052fc244b5e)